### PR TITLE
Add support for `buildx`

### PR DESCRIPTION
**Pull request by nik-kashi**
It enables to create of cross-platform images
For using the cross-platform builds, the `platforms` property should have been valued

### CODE REVIEW

File: src/main/scala/sbtdocker/DockerBuild.scala [hunk 1]
```diff
@@ -113,19 +113,23 @@ object DockerBuild {
...
       val command: Seq[String] = dockerPath ::
+        buildX :::
         "build" ::
         buildOptionFlags :::
         buildKitFlags :::
         buildArgumentFlags :::
+        load :::
         "--file" ::
-        dockerfilePath.name ::
+        dockerfileAbsolutePath.getPath ::
```
Comment (nik-kashi) [inline src/main/scala/sbtdocker/DockerBuild.scala:132]: Using an absolute path makes the command more readable and portable

File: src/main/scala/sbtdocker/DockerBuild.scala [hunk 2]
```diff
@@ -113,19 +113,23 @@ object DockerBuild {
     var lines = Seq.empty[String]
 
     def runBuild(buildKitSupport: Boolean): Int = {
+      val buildX = if (buildOptions.platforms.isEmpty) Nil else List("buildx")
+      val load = if (buildOptions.platforms.isEmpty) Nil else List("--load")
```
Comment (nik-kashi) [inline src/main/scala/sbtdocker/DockerBuild.scala:117]: The `--load` parameter is needed for `buildx` to create the image

**Comment by marcuslonnberg** [comment]
Looks good, thanks! 👏 
I'll try to make a release in the next day.

File: src/main/scala/sbtdocker/DockerBuild.scala (hunk 2 above)
Comment (kalanzai) [reply src/main/scala/sbtdocker/DockerBuild.scala:117]: Doesn't the `--load` flag only work for single-platform builds? 
From [docker docs](https://docs.docker.com/engine/reference/commandline/buildx_build/#output):
> Currently, multi-platform images cannot be exported with the docker export type. The most common usecase for multi-platform images is to directly push to a [registry](https://docs.docker.com/engine/reference/commandline/buildx_build/#registry) (see registry).

Comment (nik-kashi) [reply src/main/scala/sbtdocker/DockerBuild.scala:117]: Pushing directly to a registry makes breaking changes in this library. Feel free to create a PR that supports multiple platforms in one build

**Pull request merged**